src/handlers/massiveeos/MassiveEosActionHandler.ts

```diff
@@ -0,0 +1,31 @@
+import { HandlerVersion } from 'demux'
+import { MassiveActionHandler, MigrationSequence } from 'demux-postgres'
+
+export class MassiveEosActionHandler extends MassiveActionHandler {
+  constructor(
+    protected handlerVersions: HandlerVersion[],
+    protected massiveInstance: any,
+    protected dbSchema: string = 'public',
+    protected migrationSequences: MigrationSequence[] = [],
+  ) {
+    super(handlerVersions, massiveInstance, dbSchema, migrationSequences)
+  }
+
+  /**
+   * This method is used when matching the types of incoming actions against the types the `Updater`s and `Effect`s are
+   * subscribed to. When this returns true, their corresponding functions will run.
+   *
+   * By default, this method tests for direct equivalence between the incoming candidate type and the type that is
+   * subscribed. Override this method to extend this functionality (e.g. wildcards).
```

Comment:
You can update this comment just to mention that we're overriding to provide wildcard functionality. I don't think we need the rest of the comment.

Reply:
Good call!